packages/augur-ui/src/modules/alerts/actions/set-alert-text.ts

```diff
@@ -223,8 +220,8 @@ export default function setAlertText(alert: any, callback: any) {
             );
             const outcomeDescription =
               outcome === null
-                ? 'Market Is Invalid'
-                : getOutcomeName(marketInfo, { id: outcome }, false);
+                ? "Market Is Invalid"
```

Comment:
I thought the standard was use single quotes for strings.

Reply:
not sure what happened here